app/code/Magento/Sales/Test/Mftf/ActionGroup/AssertAdminFieldsFilledDefaultBillingAddressActionGroup.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminFieldsFilledDefaultBillingAddressActionGroup">
+        <annotations>
+            <description>Admin assert fields filled default billing address from customer</description>
+        </annotations>
+        <arguments>
+            <argument name="firstName" type="string" defaultValue="John"/>
+            <argument name="lastName" type="string" defaultValue="Doe"/>
+            <argument name="company" type="string" defaultValue="Magento"/>
+            <argument name="streetLine1" type="string" defaultValue="7700 West Parmer Lane"/>
+            <argument name="streetLine2" type="string" defaultValue="113"/>
+            <argument name="city" type="string" defaultValue="Los Angeles"/>
+            <argument name="postcode" type="string" defaultValue="90001"/>
+            <argument name="phone" type="string" defaultValue="512-345-6789"/>
+            <argument name="vatNumber" type="string" defaultValue="U1234567891"/>
```

Comment:
Please add all address fields like country, region and others
Check both possibilities for region (dropdown an plain text field)